File: Java/Algorithms/Math/Fibonacci.java
/**
 * Class Fibonacci
 */
public class Fibonacci {

    /**
     * Calculates the Fibonacci number at the given index using iteration
     *
     * @param index a given index value
     * @return int
     */
    public int fibonacciIteration(int index) {
        int a = 0;
        int b = 0;
        int c = 1;

        for (int i = 0;i < index;i++) {
            a = b;
            b = c;
            c = a + b;
        }

        return c;
    }
}

File: PHP/Algorithms/Math/fibonacci.php
<?php

/**
 * Calculates the Fibonacci number at the given index using iteration
 *
 * @param int $index
 * @return int
 */
function fibonacciIteration($index) {
    $a = 0;
    $b = 0;
    $c = 1;

    for ($i = 0;$i < $index;$i++) {
        $a = $b;
        $b = $c;
        $c = $a + $b;
    }

    return $c;
}
